modules/persistent-storage-byok.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-azure.adoc
+// storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+// storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_content-type: CONCEPT
+[id="byok_{context}"]
+= User-managed encryption
+
+The user-managed encryption feature allows you to provide keys during installation that encrypt {product-title} node root volumes, and enables all managed storage classes to use these keys to encrypt provisioned storage volumes. You must specify the custom key in the `platform.<cloud_type>.defaultMachinePlatform` field in the install-config YAML file.
+
+This features supports the following storage types:
+
+* Amazon Web Services (AWS) Elastic Block storage (EBS)
+
+* Microsoft Azure Disk storage
+
+* Google Cloud Platform (GCP) persistent disk (PD) storage
+
+// Procedure on how to make this happen. Azure is covered in Azure assembly. AWS and GCP???
```

Comment:
This comment looks like it could be removed now

Reply:
Yes. Done.